common/flatpak-dir.c

```diff
@@ -7808,12 +8050,30 @@ flatpak_dir_install (FlatpakDir          *self,
                                  flatpak_flags,
                                  OSTREE_REPO_PULL_FLAGS_MIRROR,
                                  progress, cancellable, error))
-            return FALSE;
+            {
+              if (revokefs_fuse)
+                {
+                  g_autofree gchar *repo_path = NULL;
+
+                  repo_path = g_file_get_path (ostree_repo_get_path (child_repo));
```

Comment:
Why is this needed? 14 lines later you assert that `child_repo_path` is non-NULL.